webapp/src/ts/app.module.ts

```diff
@@ -46,7 +46,9 @@ import { reducers } from '@mm-reducers/index';
 
 const logger = reducer => {
   // default, no options
-  return storeLogger()(reducer);
+  return storeLogger({
+    collapsed: true,
+  })(reducer);
```

Comment:
Accidentally committed?

Reply:
Intentional. I had been doing that and reverting for a few weeks.
It reduces the clutter, by collapsing every redux store event. The info is still there if expanded. 

![Screenshot_20210917_090405](https://user-images.githubusercontent.com/35681649/133732636-e2a01132-6345-4b67-b583-74593ef545d7.png)